packages/core/contracts/EscrowFactory.sol

```diff
@@ -34,27 +34,57 @@ contract EscrowFactory is OwnableUpgradeable, UUPSUpgradeable {
 
     function createEscrow(
```

Comment:
fund the escrow is missing

Reply:
Do we want to fund the escrow by job launcher? I think escrow can be funded from job requesters after the escrow is created?